### PR TITLE
blur: Add saturation option

### PR DESCRIPTION
**Pull request by soreau**
This option controls the saturation of the blurred image.

### CODE REVIEW

File: plugins/blur/blur-base.cpp [hunk 1]
```diff
@@ -232,6 +244,7 @@ void wf_blur_base::render(wf::texture_t src_tex, wlr_box src_box,
     /* Blend blurred background with window texture src_tex */
     blend_program.uniformMatrix4f("mvp", glm::inverse(target_fb.transform));
     /* XXX: core should give us the number of texture units used */
+    blend_program.uniform1f("sat", saturation_opt);
```
Comment (ammen99) [inline plugins/blur/blur-base.cpp:247]: please move this before the comment, because the comment is related to the next line

**Review by ammen99 — APPROVED** [review]
LGTM with one small nitpick

File: plugins/blur/blur-base.cpp (hunk 1 above)
Comment (soreau) [reply plugins/blur/blur-base.cpp]: Ah, nice catch.

**Comment by ammen99** [comment]
Thanks!

**Pull request merged**